GUI-Lib/src/main/java/de/muenchen/vaadin/guilib/components/GenericGrid.java

```diff
@@ -563,6 +563,9 @@ private I18nResolver getResolver(){
         return BaseUI.getCurrentI18nResolver();
     }
 
+    public Grid getGrid(){
```

Comment:
D.h. in der Produktion muss ich wenn ich eine Einstellung brauch entweder die LIB für alle Ändern oder die GenericGrid komplett selber schreiben und nach bauen?